tcms/static/js/testrun_actions.js

```diff
@@ -60,15 +61,25 @@ Nitrate.TestRuns.Details.on_load = function() {
   var toggle_case_run = function(e) {
     var c = jQ(this).parent(); // Container
     var c_container = c.next(); // Content Containers
-    var case_id = c.find('input[name="case"]')[0].value;
-    var case_run_id = c.find('input[name="case_run"]')[0].value;
-    var case_text_version = c.find('input[name="case_text_version"]')[0].value;
-    var type = 'case_run';
+    var caseId = c.find('input[name="case"]')[0].value;
+    var executionId = c.find('input[name="case_run"]')[0].value;
+    var caseTextVersion = c.find('input[name="case_text_version"]')[0].value;
     var callback = function(t) {
       // Observe the update case run stauts/comment form
-      c_container.parent().find('.update_form')
-        .unbind('submit').bind('submit', updateCaseRunStatus);
 
+      const updateForm = c_container.find('.update_form')
+
+      updateForm.unbind('submit').bind('submit', event => {
+        event.stopPropagation();
+        event.preventDefault();
+
+        const commentTextArea = updateForm.find('td > textarea[name=comment]')
```

Comment:
this causes Submit button to not work because the textarea selector is wrong. There's no td element surrounding it.